README.md

```diff
@@ -346,6 +346,24 @@ Clones all repositories available to a user on Bitbucket.
     - Create an application password within your [account settings](https://bitbucket.org/account/admin/app-passwords).
     - We need the scope: Repositories -> Read
 
+
+### `all_repos.source.bitbucket_server`
+
+Clones all repositories available to a user on Bitbucket Server.
+
+#### Required `source_settings`
+
+- `base_url`: the bitbucket server URL (eg `bitbucket.domain.com`)
```

Comment:
I think you can modify the existing `bitbucket` one to add `base_url` as an optional value which defaults to public bitbucket?  that would be preferable

Reply:
It would. Unfortunately the paging API is different, that part would also need to support two different versions.
Is that a direction you want to go in?

Reply:
ugh why would they have separate implementations -- that's frustrating!  this is probably fine then I'll review this later